src/common/ms-word-helpers.ts

```diff
@@ -0,0 +1,39 @@
+import { OutputData as RichTextData } from '@editorjs/editorjs';
+import { FieldInputProps } from 'react-final-form';
+import { Nullable } from './types';
+
+export const handleMsUnorderedList = (
```

Comment:
Could you please clarify a bit?

Reply:
Yeah instead of doing it all here, reduce the function to the single responsibility of parsing the data. Something like:
```ts
useEventListener('paste', (e: ClipboardEvent) => {
  e.preventDefault();
  const newValue = parsePasteData(e);
  input.onChange(newValue);
})
```

Reply:
yes, though we do need to also handle the case where the intercepted paste is not changed

Reply:
It looks like we need to conditionally prevent default and write the input (if the matcher is not triggered we should just do the normal paste) so the block can be built